src/test/java/tqs/example/impostor/repository/LockerRepositoryTest.java

```diff
@@ -0,0 +1,59 @@
+package tqs.example.impostor.repository;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.orm.jpa.DataJpaTest;
+import org.springframework.boot.test.autoconfigure.orm.jpa.TestEntityManager;
+
+import java.util.Optional;
+
+import static org.assertj.core.api.FactoryBasedNavigableListAssert.assertThat;
+
+@DataJpaTest
+public class LockerRepositoryTest {
+
+    @Autowired
+    private LockerRepository lockerRepository;
+
+    @Autowired
+    private TestEntityManager entityManager;
+
+    @Test
+    public void whenFindLockerByExistingId_thenReturnLocker() {
```

Comment:
Although the naming in this case is quite clear, please consider giving it a name such as: "givenId_whenFindById_thenReturnOptionalOfLocker".
As a sidenote this test methods testing the "findById" function won't be necessary because LockerRepository already extends JpaRepository. This is further explained in my comment in the LockerRepository.